cadc-file-validate/README.md

```diff
@@ -62,12 +62,6 @@ docker build -t opencadc/cadc-file-validate -f Dockerfile .
 docker run -t opencadc/cadc-file-validate:latest
 ```
 
-### With usage
-```
-docker run -t opencadc/cadc-file-validate:latest cadc-file-validate -h
-```
-
-
 ## Running it
```

Comment:
The cadc-java container supports running as nobody; please add/recommend --user nobody:nobody